src/test/resources/features/TestExcludeDate.feature

```diff
@@ -0,0 +1,87 @@
+    And I select No for additional support needed
+    And click on continue button
+    Then Im on "Manage prison visits - Who is the main contact for this booking?" page
+    And I select the main contact option
+    And I enter "<phoneNumber>" to get text message
+    And click on continue button
+    Then Im on "Manage prison visits - How was this booking requested?" page
+    And I select a phone call option on method used to make the reqest
+    And click on continue button
+    Then Im on "Manage prison visits - Check the visit details before booking" page
+    And click on continue button
+    Then Im on "Manage prison visits - Booking confirmed" page
+    And I see "Booking confirmed" message displayed
+    And I see a booking reference
+    # exclude date
+    Given A booking reference date is excluded
+    And I goto home page
+    Then Im on "Manage prison visits - Manage prison visits" page
+    And I click on Need a Review option
+    And Im on "Manage prison visits - Visit bookings that need review" page
+    And I click on view link to see details of the booking needs review
+    And I choose Booking history tab
+    And I see "Reason: Time slot removed" time slot removed reason
+    Then I click on update booking button
+    And Im on "Manage prison visits - Select visitors from the prisoner’s approved visitor list" page
+    Then I see "A new visit time should be selected as the date is no longer available for social visits." message is displayed
+    And I select visitor from the list
+    # updating the booking
+    And click on continue button
+    Then Im on "Manage prison visits - Select date and time of visit" page
+    And I select time slot
+    And click on continue button
+    Then Im on "Manage prison visits - Is additional support needed for any of the visitors?" page
+    And I select No for additional support needed
+    And click on continue button
+    Then Im on "Manage prison visits - Who is the main contact for this booking?" page
+    And I select the main contact option
+    And I enter "<phoneNumber>" to get text message
+    And click on continue button
+    Then Im on "Manage prison visits - How was this booking requested?" page
+    And I select a phone call option on method used to make the reqest
+    And click on continue button
+    Then Im on "Manage prison visits - Check the visit details before booking" page
+    And click on continue button
+    Then Im on "Manage prison visits - Booking confirmed" page
+    And I see "Booking confirmed" message displayed
+    And I see a booking reference
+    #check Needs A Review
+    And I goto home page
+    Then Im on "Manage prison visits - Manage prison visits" page
+    And I click on Need a Review option
+    And Im on "Manage prison visits - Visit bookings that need review" page
+    And I check "<prisonNumber>" is not available
+   # And I see booking reference is not available
```

Comment:
ideal would be to check if booking reference is available. I presume there will be some work done to check this later.

Reply:
There is no reference number on that page thats why I changed to Prisoner number instead.

Reply:
In that case it would be ideal to check for the prison number and date combination.

Reply:
Yes I'm going to add the date from this morning's conversation I picked that. It will be done in next PR.

Reply:
ok

Reply:
I will do the changes today - might be better to update the same PR instead of creating a new one.